src/internal_events/logplex.rs

```diff
@@ -20,7 +20,6 @@ impl<'a> InternalEvent for HerokuLogplexRequestReceived<'a> {
             drain_token = %self.drain_token,
             internal_log_rate_limit = true
         );
```

Comment:
Unrelated to the below change, but should the above really be an info log? It should probably be a debug log in my opinion.

Reply:
Yeah, agreed, it is a bit inconsistent. I'll update it.